libs/island-ui/core/src/lib/Input/Input.tsx

```diff
@@ -128,7 +129,6 @@ export const Input = forwardRef(
     const mergedRefs = useMergeRefs(inputRef, ref || null)
 
     const InputComponent = textarea ? TextareaHOC : InputHOC
-
```

Comment:
If this is reverted, we can probably remove this file from the changeset... that probably doesn't matter 😄  just by the way if you are making changes to this file